requirements.txt

```diff
@@ -5,3 +5,6 @@ Pillow=7.1.1
 scikit-image=0.16.2
 matplotlib=3.2.1
 tensorflow
```

Comment:
@sd2001 I am not able to view the files for IQA. I tried going through your repo. But that didn't help either. Can you please look into this

Reply:
Sure. I'll check what went wrong.